services/account/apps/api/src/controllers/v1/accounts-v1.controller.ts

```diff
@@ -69,6 +69,7 @@ export class AccountsControllerV1 {
       const response = await this.accountsService.getMsaIdForPublicKey(publicKey);
       if (response?.msaId) {
         const account = await this.accountsService.getAccount(response.msaId);
+        this.logger.debug(account, 'Account');
```

Comment:
nit: If we keep this debug, maybe make it more traceable
```suggestion
        this.logger.debug(account, 'getAccountForPublicKey:account');
```